src/shim/econ.rs

```diff
@@ -248,3 +255,18 @@ impl Sub<TokenAmount> for &TokenAmount {
         (&self.0).sub(&rhs.0).into()
     }
 }
+
+#[cfg(test)]
```

Comment:
I'm sure we need those assertions under a test module in tests. Can we have them just below the imports?